spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature 'Post index page', type: :feature do
+  let(:user_ids) { User.pluck(:id) }
+  let(:post) { Post.first }
+  before do
+    visit user_posts_path(user_id: user_ids.last)
+  end
+
+  scenario 'Displays details, posts and comments to the post for a single user' do
+    sleep(1)
+    expect(page).to have_content('Mancoba')
+    expect(page).to have_css("img[src='https://avatars.githubusercontent.com/u/53082877?s=96&v=4']")
+    expect(page).to have_content('Mancoba : Good review!')
+    expect(page).to have_css('li.user', count: 1)
+    expect(page).to have_css('li.post p.comment-heading', text: 'Comments: 7, Likes: 0')
+    expect(page).to have_css('li.post', count: 5)
+    expect(page).to have_css('p.comment', count: 5)
+  end
+
+  scenario 'Clicking on a see more post redirects to index page for posts' do
+    click_button 'Add New Post'
+    sleep(1)
+    expect(current_path).to eq(new_user_post_path(user_id: user_ids.last))
+  end
+
+  scenario 'Should create a new post' do
+    @post = Post.new(title: 'New Post Title', text: 'This is the new post text.')
+    click_button 'Add New Post'
+    sleep(1)
+    fill_in 'Title', with: @post.title
+    fill_in 'Text', with: @post.text
+    click_button 'Add Post'
+    expect(page).to have_css('li.post', count: 6)
```

Comment:
As shown in the image below, some of the tests are failing, kindly check and ensure that all the tests passed in order to produce a good and up to standard app. 
NB: this applies to all the tests in the app.


![Screenshot from 2023-03-08 12-00-43](https://user-images.githubusercontent.com/65233860/223709335-cfe82df1-57ef-44ec-8055-ab77ab47a701.png)


![Screenshot from 2023-03-08 12-01-01](https://user-images.githubusercontent.com/65233860/223709352-4cd2e352-cc92-44c1-980b-4abe39302535.png)